.eslintrc.json

```diff
@@ -5,13 +5,13 @@
   "env": {
     "browser": false,
     "node": true,
-    "jest": true,
     "es6": true
   },
   "rules": {
     "strict": 0,
-    "import/no-extraneous-dependencies": [2, {
-      "devDependencies": ["**/*.spec.js"]
-    }]
+    "id-length": 2,
```

Comment:
That's not needed since it's already included: https://github.com/okonet/eslint-config-okonet/blob/master/index.js#L19